src/main/java/it/gov/pagopa/common/mongo/config/MongoHealthConfig.java

```diff
@@ -0,0 +1,29 @@
+package it.gov.pagopa.common.mongo.config;
+
+import it.gov.pagopa.common.config.CustomMongoHealthIndicator;
+import it.gov.pagopa.common.config.HealthIndicatorLogger;
+import org.springframework.boot.actuate.health.HealthIndicator;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.mongodb.core.MongoTemplate;
+
+import java.util.List;
+
+@Configuration
+public class MongoHealthConfig    {
+    @Bean
+    public CustomMongoHealthIndicator customMongoHealthIndicator(MongoTemplate mongoTemplate) {
+        return new CustomMongoHealthIndicator(mongoTemplate);
+    }
+
+    @Bean
+    public HealthIndicatorLogger healthIndicatorLogger(List<HealthIndicator> healthIndicatorList) {
```

Comment:
Is this `HealthIndicatorLogger` still useful? I mean, K8S also reports if a pod fails due to a failed health check. Also this indicator will generate a lot of logs (sent to appinsights every x seconds).